ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java

```diff
@@ -20,7 +20,8 @@
     private static final By MANAGE_MODULES = By.cssSelector("#legacyui-manageModules a");
     private final static By MANAGE_VISIT_TYPES = By.cssSelector("#legacyui-manageVisitTypes a");
     private static final By MANAGE_PROVIDERS = By.cssSelector("#content a[href='/openmrs/admin/provider/index.htm']");
-    private static final By MANAGE_PERSONS = By.cssSelector("#content a[href=\"/openmrs/admin/person/index.htm\"]");
+    private static final By MANAGE_PERSONS = By.cssSelector("#content a[href='/openmrs/admin/person/index.htm']");
+    private static final By MANAGE_HTMLFORMS_PAGE = By.cssSelector("#content a[href*='/module/htmlformentry/htmlForms.list']");
```

Comment:
any reason why you need the asteric after href? you can also get rid of #content

Reply:
Resolved thanks

Reply:
when you inspect that `a` element, does it have a full link?

Reply:
Yes it have the full link like **<a href=" /openmrs/module/htmlformentry/htmlForms.list ">Manage HTML Forms</a>**, but removing the asterisk breaks the automation